src/middlewared/middlewared/plugins/catalogs_linux/items.py

```diff
@@ -38,9 +38,11 @@ def items(self, label, options):
         elif not os.path.exists(catalog['location']):
             self.middleware.call_sync('catalog.update_git_repository', catalog, True)
 
+        self.middleware.call_sync('alert.oneshot_delete', 'CatalogNotHealthy', f'"{label}"')
```

Comment:
Why a f-string?

Reply:
Ah, the quotes, nevermind, but why quotes?

Reply:
So basically when we create an alert specifying `key`, it json dumps the value which results in the key having quotes and thus we need to do the same during deletion. But if we use `_key` instead, that does not happen and i changed that. Thank you for pointing it out :)